build/swagger_config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
```

Comment:
Should we change as well the `build/package.json` to match the new swagger version?